src/ontology/reports/gard-mondo-mapped-obsoletes.tsv

```diff
@@ -0,0 +1,56 @@
+subject_id	subject_label	predicate_id	object_id	object_label	mapping_justification	subject_source	object_source
```

Comment:
I'm not sure that this file (`reports/gard-mondo-mapped-obsoletes.tsv`), nor `reports/obsoletes.tsv` are correct because I'm not sure I'm using the right `mondo.owl`. See [my comment in the issue](https://github.com/monarch-initiative/mondo/issues/6436#issuecomment-1638798061) to see how I got it.

Note that this has the same number of cases as when I used `mondo-edit.owl`, 55. Not the expected 785.

Reply:
outdated. have new output